scripts/validation/lib/changed-files.cjs

```
@@ -0,0 +1,55 @@
+  if (!ref) return null;
+  try {
+    return git(["rev-parse", "--verify", "--quiet", `${ref}^{commit}`]);
+  } catch {
+    // Also covers the all-zero SHA GitHub sends as github.event.before on the
+    // first push to a branch, which resolves to nothing.
+    return null;
+  }
+}
+
+function resolveRange() {
+  const baseSha = commitSha(process.env.BASE_SHA);
```

ESLint

[error] 'process' is not defined.

(no-undef)